src/apps/material/material.tsx

```diff
@@ -239,7 +239,11 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
         disclosureIconExpandAltText=""
         dataCy="material-details-disclosure"
       >
-        <MaterialDetailsList className="pl-80 pb-48" data={detailsListData} />
+        <MaterialDetailsList
+          id="material-details"
```

Comment:
Consider adding the `wid` (work id) to the `id` string.

This helps to ensure uniqueness even if different material apps are used on the same page.